### PR TITLE
Add option for ask the type of meeting

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -59,6 +59,10 @@
                 "help_text": "How should the plugin choose a meeting name? Some names may cause conflicts if you use them on a public server, like meet.jit.si.",
                 "default": "english-titlecase",
                 "options": [
+                    {
+                        "display_name": "Ask for meeting type when you start a meeting",
```
Comment (jespino) [inline plugin.json:63]: @jasonblais maybe you want to rephrase this option.

File: server/plugin.go [hunk 2]
```diff
@@ -234,9 +231,88 @@ func (c Claims) MarshalBinary() (data []byte, err error) {
...
+				"meeting_id":    generateUUIDName(),
+				"meeting_topic": "Jitsi Meeting",
+				"personal":      false,
+			},
+		},
+	})
+
+	sa := model.SlackAttachment{
+		Title:   "Jitsi Meeting Start",
+		Text:    "Please select the type of meeting that you want to start",
```
Comment (jasonblais) [inline server/plugin.go:304]:
```suggestion
		Text:    "Select type of meeting you want to start",
```

File: server/plugin.go [hunk 3]
```diff
@@ -234,9 +231,88 @@ func (c Claims) MarshalBinary() (data []byte, err error) {
...
+
+	actions := []*model.PostAction{}
+
+	var team *model.Team
+	if channel.TeamId != "" {
+		team, _ = p.API.GetTeam(channel.TeamId)
+	}
+
+	actions = append(actions, &model.PostAction{
+		Name: "Random Words",
```
Comment (jasonblais) [inline server/plugin.go:253]:
```suggestion
		Name: "Meeting name with random words",
```

Comment (jasonblais) [reply server/plugin.go:253]: ^0/5, but `Random Words` may not be obvious.

File: server/plugin.go [hunk 4]
```diff
@@ -234,9 +231,88 @@ func (c Claims) MarshalBinary() (data []byte, err error) {
...
+					"meeting_id":    generateTeamChannelName(team.Name, channel.Name),
+					"meeting_topic": fmt.Sprintf("%s Channel Meeting", channel.DisplayName),
+					"personal":      false,
+				},
+			},
+		})
+	}
+
+	actions = append(actions, &model.PostAction{
+		Name: "UUID meeting",
```
Comment (jasonblais) [inline server/plugin.go:291]:
```suggestion
		Name: "Meeting name with UUID",
```

File: plugin.json (hunk 1 above)
Comment (jasonblais) [reply plugin.json:63]:
```suggestion
                        "display_name": "Allow user to select meeting name",
```

**Comment by jespino** [comment]
@jasonblais done :)

**Pull request merged**